app/lib/methods/helpers/openLink.ts

```diff
@@ -36,10 +36,17 @@ const appSchemeURL = (url: string, browser: string): string => {
 	return schemeUrl;
 };
 
+const addProtocol = (url: string): string => {
```

Comment:
Adds a test for this function:
http://www.google.com
https://www.google.com
www.google.com

Reply:
Sure!